src/models/RoomConfiguration.ts

```diff
@@ -0,0 +1,6 @@
+export interface RoomConfiguration {
+    name: string,
+    cardSchemeType: 'custom' | 'regular' | 'fibonacci',
```

Comment:
Als je in het `CardScheme` model een tagged union maakt kan je hier `CardScheme['type']` gebruiken zodat je niet ieder type hier weer handmatig toe moet voegen.